subprojects/language-java/src/main/java/org/gradle/api/tasks/compile/CompileOptions.java

```diff
@@ -311,7 +311,7 @@ public void setExtensionDirs(@Nullable String extensionDirs) {
      *
      * Compiler arguments not supported by the DSL can be added here. For example, it is possible
      * to pass the {@code --release} option of JDK 9:
-     * <pre><code>compilerArgs.addAll(['--release', '7'])</code></pre>
+     * <pre><code>compilerArgs.addAll(['--release', '9'])</code></pre>
```

Comment:
Thanks, but this isn't actually a typo.

This is trying to say "it's possible to pass the `--release` option that was added in JDK 9"